src/libraries/tests.proj

```diff
@@ -151,6 +151,10 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Private.Xml.Linq\tests\xNodeReader\System.Xml.Linq.xNodeReader.Tests.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetOS)' != 'iOS' and '$(TargetOS)' != 'tvOS'">
+    <ProjectExclusions Include="..\mono\netcore\sample\iOS\Program.csproj" />
+  </ItemGroup>
```

Comment:
we can remove this ProjectExclusions and instead check the conditions in the ProjectReference instead

Reply:
Removed, thanks!